core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -683,7 +683,7 @@ public boolean isCoordinator() {
       }
 
       try {
-         log.tracef("About to wire and start cache %s", cacheName);
+         CONTAINER.debugf("Constructing and starting cache %s", cacheName);
```

Comment:
replace `Contructing` with `Creating` sound better to me. wdyt?

Reply:
I didn't want to use `Creating` to avoid any allusion to `CacheContainerAdmin.createCache()`, which creates the cache on all the nodes.

Reply:
In the end, I went with "Creating cache %s on %s"